<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>724</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToProfile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to My Profile</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="expandedPost">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>701</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>701</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="enterComment">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>661</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="commentButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add Comment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="postBackButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>661</width>
       <height>111</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>659</width>
        <height>109</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>635</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="expandedPostText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>605</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableWidget" name="commentsTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>660</width>
       <height>111</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column/>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Comments</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="searchBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="searchEdit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>156</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>11</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>üîç</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="searchError">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>49</y>
       <width>156</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginBox" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QLabel" name="loginMessage">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>Enter your name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="loginEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>61</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>125</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>New user?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="createAcc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Create an account</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createAccBox" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
    <widget class="QLabel" name="firstMessage">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>First name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="firstEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="createAccButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Create account</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="yearEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="yearMessage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>Birth year:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="zipEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="zipMessage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>Zip code:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lastMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>Last name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="lastEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="friendsBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="friendsLabel">
       <property name="text">
        <string>Friends</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="friendsTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="removeFriendButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Friend</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="addPostBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>350</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="postEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="privateCheck">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>85</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Private</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPostButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Post</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="suggestionsBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="suggestionsLabel">
       <property name="text">
        <string>Friend Suggestions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="suggestionsTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="postsTable">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>350</width>
      <height>301</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="postsLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posts</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backToPrev">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>backToProfile</zorder>
   <zorder>logout</zorder>
   <zorder>profileLabel</zorder>
   <zorder>addFriendButton</zorder>
   <zorder>searchBox</zorder>
   <zorder>friendsBox</zorder>
   <zorder>addPostBox</zorder>
   <zorder>removeFriendButton</zorder>
   <zorder>suggestionsBox</zorder>
   <zorder>createAccBox</zorder>
   <zorder>loginBox</zorder>
   <zorder>expandedPost</zorder>
   <zorder>backToPrev</zorder>
   <zorder>postsLabel</zorder>
   <zorder>postsTable</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
